### PR TITLE
remove logrequest, logr

### PR DESCRIPTION
**Pull request by orthagonal**

### CODE REVIEW

File: test/test.rapptor.js [hunk 1]
```diff
@@ -62,11 +33,13 @@ tap.test('response method ', async t => {
   const response1 = await handler({
     path: '/api',
     method: 'get',
+    headers: {},
```
Comment (jgallen23) [inline test/test.rapptor.js:35]: Why was this added?

File: index.js [hunk 2]
```diff
@@ -7,6 +7,13 @@ let config = {
     initLog: false,
     unhandledRejection: true,
     uncaughtException: true,
+    logger(msg) {
```
Comment (jgallen23) [inline index.js:10]: Can we add a test for this? Stub out log and error functions and make sure it's routed to the right place

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: test/test.rapptor.js (hunk 1 above)
Comment (orthagonal) [reply test/test.rapptor.js:35]: A bunch of the tests break because we're now logging the user-agent and referer headers, this just fixes those tests so they have a headers field and don't crash.

File: test/test.rapptor.js [hunk 3]
```diff
@@ -106,3 +81,38 @@ tap.test('response method ', async t => {
   });
   t.end();
 });
+
+tap.test('response method ', async t => {
```
Comment (jgallen23) [inline test/test.rapptor.js]: this is testing error logs, right?
